src/viam/sdk/robot/service.cpp

```diff
@@ -205,13 +210,5 @@ std::shared_ptr<Resource> RobotService_::resource_by_name(Name name) {
     return r;
 }
 
-void RobotService_::register_server(std::shared_ptr<Server> server) {
-    server->register_service(this);
-}
-
-std::shared_ptr<RobotService_> RobotService_::create() {
```

Comment:
I didn't think `create` was necessary given the existence of the constructor. Let me know if that's not true.